draft-ietf-tls-tls13.md

```diff
@@ -2306,9 +2387,11 @@ values, each representing a single set of key agreement parameters;
 for instance a client might offer shares for several elliptic curves
 or multiple integer DH groups. The shares for each ClientKeyShareOffer
 MUST by generated independently. Clients MUST NOT offer multiple
-ClientKeyShareOffers for the same parameters. It is explicitly
-permitted to send an empty client_key_share extension as this is used
-to elicit the server's parameters if the client has no useful
+ClientKeyShareOffers for the same parameters. Servers receiving duplicate
+ClientKeyShareOffers SHOULD respond with an "illegal_parameter" alert
```

Comment:
everywhere else we're saying that the endpoint MUST send an alert and end a connection, I don't see why this should be an exception